tests/php/StoreApi/Routes/Cart.php

```diff
@@ -63,120 +63,135 @@ public function setUp() {
-		$routes = rest_get_server()->get_routes();
-		$this->assertArrayHasKey( '/wc/store/cart', $routes );
-		$this->assertArrayHasKey( '/wc/store/cart/apply-coupon', $routes );
-		$this->assertArrayHasKey( '/wc/store/cart/remove-coupon', $routes );
-		$this->assertArrayHasKey( '/wc/store/cart/update-customer', $routes );
-		$this->assertArrayHasKey( '/wc/store/cart/select-shipping-rate', $routes );
-	}
-
 	/**
 	 * Test getting cart.
 	 */
 	public function test_get_item() {
-		$response = rest_get_server()->dispatch( new \WP_REST_Request( 'GET', '/wc/store/cart' ) );
-		$data     = $response->get_data();
-
-		$this->assertEquals( 200, $response->get_status() );
-		$this->assertEquals( 3, $data['items_count'] );
-		$this->assertEquals( 2, count( $data['items'] ) );
-		$this->assertEquals( true, $data['needs_payment'] );
-		$this->assertEquals( true, $data['needs_shipping'] );
-		$this->assertEquals( '30', $data['items_weight'] );
-
-		$this->assertEquals( 'USD', $data['totals']->currency_code );
-		$this->assertEquals( 2, $data['totals']->currency_minor_unit );
-		$this->assertEquals( '3000', $data['totals']->total_items );
-		$this->assertEquals( '0', $data['totals']->total_items_tax );
-		$this->assertEquals( '0', $data['totals']->total_fees );
-		$this->assertEquals( '0', $data['totals']->total_fees_tax );
-		$this->assertEquals( '100', $data['totals']->total_discount );
-		$this->assertEquals( '0', $data['totals']->total_discount_tax );
-		$this->assertEquals( '0', $data['totals']->total_shipping );
-		$this->assertEquals( '0', $data['totals']->total_shipping_tax );
-		$this->assertEquals( '0', $data['totals']->total_tax );
-		$this->assertEquals( '2900', $data['totals']->total_price );
+		$this->assertAPIResponse(
+			'/wc/store/v1/cart',
+			200,
+			array(
+				'items_count'    => 3,
+				'needs_payment'  => true,
+				'needs_shipping' => true,
+				'items_weight'   => '30',
+				'items'          => function( $value ) {
+					return count( $value ) === 2;
+				},
+				'totals'         => array(
+					'currency_code'               => 'USD',
+					'currency_minor_unit'         => 2,
+					'currency_symbol'             => '$',
+					'currency_decimal_separator'  => '.',
+					'currency_thousand_separator' => ',',
+					'currency_prefix'             => '$',
+					'currency_suffix'             => '',
+					'total_items'                 => '3000',
+					'total_items_tax'             => '0',
+					'total_fees'                  => '0',
+					'total_fees_tax'              => '0',
+					'total_discount'              => '100',
+					'total_discount_tax'          => '0',
+					'total_shipping'              => '0',
+					'total_shipping_tax'          => '0',
+					'total_tax'                   => '0',
+					'total_price'                 => '2900',
+					'tax_lines'                   => array(),
+				),
+			)
+		);
 	}
 
 	/**
-	 * Test removing a nonexistent cart item.
+	 * Test removing a nonexistent cart item. This should return 404.
```

Comment:
This comment does not match the test on line 118. The tests pass, so it's the comment that should be updated. 404 should be returned when the _route_ doesn't exist, right?